### PR TITLE
Mostafa/paging

### PR DESCRIPTION
**Pull request by MostafaBalata**

### CODE REVIEW

File: src/icos_config/index.js [hunk 1]
```diff
@@ -2,19 +2,44 @@ import moment from 'moment';
...
+    length: icosAsList.length,
+  };
 };
 
 export const getICOsAsDict = () => {
   const icosAsDict = {};
   const icosAsList = getICOsAsList();
-  icosAsList.forEach((element) => {
-    if (element) {
+  console.log(icosAsList);
```
Comment (krzkaczor) [inline src/icos_config/index.js]: remove pls

File: src/icos_config/index.js [hunk 2]
```diff
@@ -2,19 +2,44 @@ import moment from 'moment';
...
+    if (icosAsList.icos[i] && icosAsList.icos[i].crowdSaleTokenContract === address) { return icosAsList.icos[i]; }
+  }
+  return null;
+};
+
+
+export const getNextICOAddressByPreviousAddress = (previousAddress) => {
+  const icosAsList = getICOsAsList();
+  for (let i = 0; i < icosAsList.icos.length; i += 1) {
+    console.warn(i, icosAsList.icos.length - 1);
```
Comment (krzkaczor) [inline src/icos_config/index.js]: remove pls

File: src/icos_config/index.js [hunk 3]
```diff
@@ -2,19 +2,44 @@ import moment from 'moment';
...
       icosAsDict[element.crowdSaleTokenContract] = element;
     }
   });
   return icosAsDict;
 };
+
+
+export const getICOByAddress = (address) => {
+  const icosAsList = getICOsAsList();
+  for (let i = 0; i < icosAsList.icos.length; i += 1) {
```
Comment (krzkaczor) [inline src/icos_config/index.js]: you could use _.find or array.prototype.find here

File: src/icos_config/index.js [hunk 4]
```diff
@@ -2,19 +2,44 @@ import moment from 'moment';
...
+  return null;
+};
+
+
+export const getNextICOAddressByPreviousAddress = (previousAddress) => {
+  const icosAsList = getICOsAsList();
+  for (let i = 0; i < icosAsList.icos.length; i += 1) {
+    console.warn(i, icosAsList.icos.length - 1);
+    const ico = icosAsList.icos[i];
+    if (ico && ico.crowdSaleTokenContract === previousAddress && i !== icosAsList.icos.length - 2) { return icosAsList.icos[i + 1].crowdSaleTokenContract; }
```
Comment (krzkaczor) [inline src/icos_config/index.js]: same here — find

Comment (MostafaBalata) [reply src/icos_config/index.js]: This one I'm looking for the next one.

**Pull request merged**